CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 * Cover InstanceInformation by jest/RTL tests. Refs UIREQ-950.
 * Fix inconsistency in RTL/Jest tests. Refs UIREQ-979.
 * Remove redundant ariaLabel prop. Refs UIREQ-972.
+* Request page preparation. Refs UIREQ-978.
```

Comment:
Could you please change to  "Preparation for Enable Request Policy to Determine Allowed Pickup Service Points. Refs UIREQ-978."?

Reply:
fixed